gnovm/stdlibs/std/banker.gno

```diff
@@ -93,6 +106,10 @@ func (b banker) SendCoins(from, to Address, amt Coins) {
 	if b.bt == BankerTypeReadonly {
 		panic("BankerTypeReadonly cannot send coins")
 	}
+	if b.pkgAddr != from.String() {
+		msg := "can only send coins from realm that created banker(" + b.pkgAddr + ")" + " not ( " + from.String() + ")" // TODO: format string needed
```

Comment:
```suggestion
		msg := `can only send coins from realm that created banker "` + b.pkgAddr + `", not "` + from.String() + `"`
```

Reply:
I also want to see some test cases where this error message shows up

Reply:
00c7e30631798bf2c9a921984fa1530be418dbf7